src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -222,12 +222,12 @@ internal ImmutableArray<TypeSymbolWithAnnotations> GetTypeParametersAsTypeArgume
             // Resolving [NonNullTypes] only makes sense within the definition of the generic type
             // or method. If this is a substituted symbol, we use the default NonNullTypes context.
```

Comment:
Is "default NonNullTypes context" accurate here now? It seems like we're using the false context. Or am I just misunderstanding the terminology?

Reply:
"default NonNullTypes context" seems correct, since `[NonNullTypes(false)]` is the root context (outside of the scope of an explicit `[NonNullTypes...]`).
But here it really doesn't matter which context we use for that case. I'll change to say "dummy context".

Reply:
This made me think we shouldn't even use the `NonNullTypesFalseContext`. I created one more fixed context that always throws.